test/epoch_token_locker.js

```diff
@@ -0,0 +1,216 @@
+const EpochTokenLocker = artifacts.require("EpochTokenLocker")
+const IntervalTokenStoreTestInterface = artifacts.require("IntervalTokenStoreTestInterface")
+const MockContract = artifacts.require("MockContract")
+const ERC20Interface = artifacts.require("ERC20")
+
+
+
+const truffleAssert = require("truffle-assertions")
+
+
+contract("EpochTokenLocker", async (accounts) => {
+  const [user_1] = accounts
+
+  describe("deposit", () => {
+    it("processes a deposit and stores it in the pendingDeposits", async () => {
+      const instance = await EpochTokenLocker.new()
```

Comment:
Not super fan of calling something instance. Maybe `epochTokenLocker`, since it's an instance of `EpochTokenLocker` contract.

Reply:
we usually call the unit under test instance, but I'm also fine with being more explicit.

Reply:
Ok. I think is a fair naming convention. I've seen it called "subject", instance is ok too.
I like it either way, so we can continue as you've been doing